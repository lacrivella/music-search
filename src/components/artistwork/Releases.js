import React from 'react';
import PropTypes from 'prop-types';
import Release from './Release';


function Releases({ releases, artist }) {
  const releaseList = releases.map(release => {
    return (
      <li key={release.id}>
        <Release release={release} artist={artist} />
      </li>
    );
  });
  return (
    <ul>
      {releaseList}
    </ul>
  );
}

Releases.propTypes = {
  releases: PropTypes.array.isRequired,
  artist:  PropTypes.string
};

export default Releases;
