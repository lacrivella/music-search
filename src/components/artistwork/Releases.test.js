import React from 'react';
import { shallow } from 'enzyme';
import Releases from './Releases';

describe('Releases component', () => {
  it('renders Releases', () => {
    const wrapper = shallow(<Releases releases={[{ title: 'hello', id: 'fake' }]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
