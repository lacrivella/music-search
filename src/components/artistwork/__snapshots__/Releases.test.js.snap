// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Releases component renders Releases 1`] = `
<ul>
  <li
    key="fake"
  >
    <Release
      release={
        Object {
          "id": "fake",
          "title": "hello",
        }
      }
    />
  </li>
</ul>
`;
